build/images/Dockerfile.clickhouse.monitor.ubuntu

```diff
@@ -0,0 +1,19 @@
+ARG GO_VERSION
+FROM golang:${GO_VERSION} as clickhouse-monitor-build
+
+COPY . /antrea
+WORKDIR /antrea/plugins/flow-visibility/clickhouse-monitor
+
+# Statically links clickhouse-monitor-plugin binary to make sure it can run in scratch.
```

Comment:
I feel "run in scratch" is not a phrase.  Maybe you can remove "to make sure it can run in scratch" or rephrase.

Reply:
Thanks Jianjun for reviews. Removed